roles/contiv_network/tasks/aci_tasks.yml

```diff
@@ -23,6 +23,7 @@
   service: name=aci-gw state=started
   run_once: true
 
+# NOTE: Use Contivctl command when it is available
 - name: set aci mode
-  shell: contivctl global set -nwinfra aci
```

Comment:
Just wondering what does it take to make this command available in contivctl?

I don't think netctl is supposed to be our production cli, so we shall to avoid using it in ansible as much as possible.